go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/mattn/go-isatty v0.0.16
 	github.com/mongodb-forks/digest v1.0.4
 	github.com/mongodb-labs/cobra2snooty v0.12.2
+	github.com/mongodb/mongodb-atlas-kubernetes v1.3.1-0.20221024110443-0c2d71e4f1e4
```

Comment:
Did you manage to create a GitHub team that we can tag to review the dependapot PRs to update your dependencies?

Reply:
Yes, sent you a PM